wiki/installing-and-updating-drivers.md

```diff
@@ -0,0 +1,215 @@
+
+3. Click "Driver updates" to show the list of available driver updates, if there are any.
+
+<img src="./img/installing-and-updating-drivers/optional-updates/windows-update-optional-updates-driver-updates.png" alt="The Optional updates menu, with the Driver updates menu highlighted." width="500px">
+
+4. Select any optional updates you wish to install, then click "Download & install".
+
+<img src="./img/installing-and-updating-drivers/optional-updates/windows-update-driver-update-example.png" alt="An available optional driver update, with the Download & install button selected." width="500px">
+
+On Windows 10, a "View optional updates" link will appear under "Check for updates" if optional updates are available. Click on it to see the list of available driver updates.
+
+## Method 2: Using your OEM's website
+::: tip Note
+"OEM" here refers to the manufacturer of an entire device, like a laptop or pre-built desktop. If you are using a custom-built computer, you should follow [Method 3](#method-3-using-your-manufacturer-s-website) instead.
+:::
+It is highly recommended to download or update your drivers through your manufacturer's website, as that will guarantee proper functionality, provided you've installed the correct drivers. This is especially a good idea for your Graphics Card (GPU) Drivers.
+
+All you need to do is:
+1. On the search engine of your choice (Google, Bing, etc…), search for "{device manufacturer} drivers".
+2. Visit the manufacturer's website and enter the model of your device, then follow the instructions on the page.
+3. When you select the product, you may be shown with 2 options:
+	- Download the driver downloader (recommended).
+	- Download this specific driver only and install manually.
+### Well-known manufacturers' websites
+- [Dell](https://www.dell.com/support/home/en-us?app=drivers)
+ 
+- [HP](https://support.hp.com/us-en/drivers)
+ 
+- [Lenovo](https://pcsupport.lenovo.com/us/en)
+## Method 3: Using your manufacturer's website
+::: tip Note
+"Manufacturer" here refers to the manufacturer of an individual piece of hardware. If you are using a laptop or a pre-built desktop, you should probably download drivers from your OEM's website instead. See [Method 2](#method-2-using-your-oem-s-website).
+:::
+For each piece of hardware in your computer that you wish to get drivers for:
+1. On the search engine of your choice (Google, Bing, etc…), search for "{hardware manufacturer} drivers".
+ 
+2. Search for the model number of the piece of hardware you want to upgrade/install drivers for.
+ 
+3. When you select the product, you may have two options:
+ 
+    - Download the driver downloader (recommended).
+    - Download this specific driver only and install manually.
+
+
+See [Reinstalling GPU drivers from the OEM's website](/wiki/reinstalling-gpu-drivers.html#method-3-oem-website) for more information on installing GPU drivers.
```

Comment:
Strip the `.html` for clean URLs so we don't make an extra client side redirect in case we switch to VitePress.

```suggestion
See [Reinstalling GPU drivers from the OEM's website](reinstalling-gpu-drivers#method-3-oem-website) for more information on installing GPU drivers.
```